packages/ra-ui-materialui/src/layout/ResourceMenuItem.tsx

```diff
@@ -22,7 +24,14 @@ export const ResourceMenuItem = ({ name }: { name: string }) => {
                 type: 'list',
             })}
             state={{ _scrollToTop: true }}
-            primaryText={getResourceLabel(name, 2)}
+            primaryText={
+                <>
+                    {translate(`resources.${name}.name`, {
+                        smart_count: 2,
+                        _: getResourceLabel(name, 2),
+                    })}
+                </>
+            }
```

Comment:
I don't understand this change. Doesn't `getResourceLabel()` already do the translation?

Reply:
To avoid a double translation (See MenuItemLink)